Makefile

```diff
@@ -125,4 +125,7 @@ test:
 	py.test --flake8 --cov-report term-missing --cov=leapp tests/scripts
 	py.test --flake8 leapp
 
+pep8:
+	pylint leapp tests/scripts/*.py
+
 .PHONY: clean copr_build install install-deps install-test srpm test
```

Comment:
```suggestion
.PHONY: clean copr_build install install-deps install-test srpm test pep8
```

Reply:
Sure, I'll remove tox.ini and fix phony targets list.
As for using pytest-pylint - I spent around 1 hour trying to make it not ignore anything but tests and failed, so I returned to the pure pylint. Maybe you have the idea what I was doing wrong and why py.test --pylint was ignoring project directory skipping all files there?

Reply:
The trick is to pass path to pylint, otherwise it searches for tests. So `py.test --pylint --flake8 leapp` works for me, it invokes pylint and flake8 checks on all `*.py` files in the `leapp` folder. Also if pytest is used for invoking linters, all configuration can be stored in one file, for both tests, and linters - `setup.cfg`. So even `.flake8` config could be deleted then.